app/views/components/fields/FieldContentFormatter.scala

```diff
@@ -21,8 +21,8 @@ import java.time.{Instant, ZoneId}
 
 object FieldContentFormatter {
 
-  private val dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm").withZone(ZoneId.systemDefault())
+  private val dateTimeFormatter = DateTimeFormatter.ofPattern("dd MMM yyyy - HH:mm").withZone(ZoneId.systemDefault())
```

Comment:
you can put 'at' instead - and replace

Reply:
doesn;t work. its not a valid pattern

Reply:
"dd MMM yyyy 'at' HH:mm" - something like this